1forge/test/adapter.test.ts

```diff
@@ -1,9 +1,13 @@
-const { assert } = require('chai')
-const { assertSuccess, assertError } = require('@chainlink/adapter-test-helpers')
-const { execute } = require('../adapter')
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { makeExecute } from '../src/adapter'
+import { AdapterRequest } from '@chainlink/types'
 
 describe('execute', () => {
   const jobID = '1'
+  const execute = makeExecute()
+  process.env.API_KEY = process.env.API_KEY ?? 'test_api_key'
```

Comment:
I don't like this a lot, setting directly the process.env field, and with a value that is not providing any actual functionality...

But we could tackle this separately.

Reply:
Agreed. This would be out of scope of this TS migration though.

There is an issue tracking it @ #245.